.github/workflows/macos_python.yml

```diff
@@ -46,6 +46,7 @@ jobs:
     - name: spack install
       run: |
         . .github/workflows/install_spack.sh
+        brew install glfw
         spack config add packages:opengl:paths:opengl@4.1:/usr/X11R6
```

Comment:
The path (and version) will probably change. Can we also switch to `spack external find opengl`? I added support for that a while back.